debs/generate_debian_package.sh

```diff
@@ -144,16 +149,26 @@ main() {
             ;;
         "-t"|"--target")
             if [ -n "$2" ]; then
-                TARGET="$2"
-                shift 2
+                if [[ "$2" == "agent" ]] || [[ "$2" == "manager" ]] || [[ "$2" == "api" ]]; then
+                    TARGET="$2"
+                    shift 2
+                else
+                    echo "Target must be manager, agent or api"
+                    help 1
+                fi
             else
                 help 1
             fi
             ;;
         "-a"|"--architecture")
             if [ -n "$2" ]; then
-                ARCHITECTURE="$2"
-                shift 2
+                if [[ "$2" == "x86_64" ]] || [[ "$2" == "amd64" ]]; then
```

Comment:
This check should be done in the build function. In addition, this check is wrong. The script support amd64, i386 and ppc64le as architecture. Check it on the build function.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/255/commits/ac22541a388ac6ed3a45b864fe89d04baccd4d20

Reply:
This isn't resolved yet.